src/sentry/templatetags/sentry_assets.py

```diff
@@ -35,3 +35,17 @@ def crossorigin():
         # They share the same domain prefix, so we don't need CORS
         return ''
     return ' crossorigin="anonymous"'
+
+
+@register.simple_tag
+def locale_js_include(request):
+    """
+    If the user has a non-English locale set, returns a <script> tag pointing
+    to the relevant locale JavaScript file
+    """
+    lang_code = request.LANGUAGE_CODE
+    if lang_code == 'en' or lang_code not in settings.SUPPORTED_LANGUAGES:
+        return ''
+
+    href = get_asset_url("sentry", "dist/moment/locale/" + lang_code + ".js")
+    return "<script src=\"{0}\" {1}></script>".format(href, crossorigin())
```

Comment:
Remove the space here before `{1}`. `crossorigin()` spits out results prefixed with a space.